faq/theme.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Theme FAQ
+---
+
+# Theme FAQ
+
+**Q:** Why new-theme directory does not have a package.json?
+
+**A:** If you use PrestaShop's production ZIP release instead of the sources, new-theme directory will not have a package.json. `npm run build` will then fail.
+
+```
+Module build failed: Error: Couldn't find preset "env" relative to directory
+```
+
+You can solve this by changing to a release for development or by copy the necessary json file to your new-theme folder, navigate there and run `npm install`.
```

Comment:
IMHO, the production build shouldn't be tweaked. But the source must be used instead

Reply:
```suggestion
PLEASE DON'T CHANGE THE PRODUCTION ZIP, USE SOURCE AND CREATE A RELEASE IF NEEDED!
```

Reply:
@PierreRambaud I think I can guess what you mean 😄